### PR TITLE
Update Filecoin chain configuration

### PR DESCRIPTION
**Pull request by navFooh**
## Description

- For Filecoin Mainnet, it is also advised to use the v1 RPC url. When the FEVM launches on Mainnet, v1 is required to use the `eth_` RPC methods.
- The Glif block explorer was missing from the Mainnet chain.
- The Calibrationnet configuration was missing
- The `glif` key was misspelled in the `blockExplorers` object for Hyperspacenet

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

### CODE REVIEW

File: packages/chains/src/filecoinCalibration.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    symbol: 'tFIL',
+  },
+  rpcUrls: {
+    default: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
+    public: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
+  },
+  blockExplorers: {
+    default: { name: 'Glif', url: 'https://explorer.glif.io/?network=calibration' },
+  },
+}
```
Comment (tmm) [inline packages/chains/src/filecoinCalibration.ts:19]:
```suggestion
export const filecoinCalibration = {
  id: 314_159,
  name: 'Filecoin Calibration',
  network: 'filecoin-calibration',
  nativeCurrency: {
    decimals: 18,
    name: 'testnet filecoin',
    symbol: 'tFIL',
  },
  rpcUrls: {
    default: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
    public: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
  },
  blockExplorers: {
    default: { name: 'Glif', url: 'https://explorer.glif.io/?network=calibration' },
  },
} as const satisfies Chain
```

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a path [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning) and export the new chains from the `packages/chains/src/index.ts` file.

Also, looks like the Glif block explorer doesn't follow [EIP-3091](https://eips.ethereum.org/EIPS/eip-3091). Would be great if the `url` could be a base URL so folks can add on different paths (e.g. `/address`, `/tx`). Right now, with the query param at the end, this won't work by default. Seems like switching to Filescan or another could be the way to go (e.g. `https://calibration.filscan.io` and `https://hyperspace.filscan.io`).

File: packages/chains/src/filecoinCalibration.ts (hunk 1 above)
Comment (navFooh) [reply packages/chains/src/filecoinCalibration.ts]: Hi @tmm, thank you for the feedback!

Actually, Glif does support the EIP-3091 standard:
- https://explorer.glif.io/calibration/address/t01/
- https://explorer.glif.io/tx/bafy2bzaceayqt5cfkcgmse4ubezbfh2nl2336tmklengcxohtf5mrynalqoum/?network=calibration

Although currently there is no support for the `/block` and `/token` routes.

File: .changeset/old-adults-jump.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': patch
+---
+
+Update Filecoin chains (use v1 RPC api, add Calibrationnet)
```
Comment (tmm) [inline .changeset/old-adults-jump.md:5]:
```suggestion
Updated Filecoin Chains. Added Filecoin Calibration Chain.
```

File: packages/chains/src/filecoinCalibration.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    decimals: 18,
+    name: 'testnet filecoin',
+    symbol: 'tFIL',
+  },
+  rpcUrls: {
+    default: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
+    public: { http: ['https://api.calibration.node.glif.io/rpc/v1'] },
+  },
+  blockExplorers: {
+    default: { name: 'Glif', url: 'https://explorer.glif.io/calibration' },
```
Comment (tmm) [inline packages/chains/src/filecoinCalibration.ts:17]:
```suggestion
    default: { name: 'Filscan', url: 'https://calibration.filscan.io' },
```

File: packages/chains/src/filecoinHyperspace.ts [hunk 4]
```diff
@@ -15,6 +15,6 @@ export const filecoinHyperspace = {
   },
   blockExplorers: {
     default: { name: 'Filfox', url: 'https://hyperspace.filfox.info/en' },
-    gilf: { name: 'Glif', url: 'https://explorer.glif.io/?network=hyperspace' },
+    glif: { name: 'Glif', url: 'https://explorer.glif.io/hyperspace' },
```
Comment (tmm) [inline packages/chains/src/filecoinHyperspace.ts:18]:
```suggestion
    filscan: { name: 'Filscan', url: 'https://hyperspace.filscan.io' },
```

**Review by tmm — APPROVED** [review]

**Comment by navFooh** [comment]
@tmm thanks for getting this over the finish line.

Out of curiosity, what's the motivation for replacing Glif with Filscan a calibration / hyperspace explorer?

Glif does support the EIP-3091 standard where Filscan doesn't:
- ✅ https://explorer.glif.io/calibration/address/t01
- ✅ https://explorer.glif.io/calibration/tx/bafy2bzaceayqt5cfkcgmse4ubezbfh2nl2336tmklengcxohtf5mrynalqoum
- ⛔ https://calibration.filscan.io/address/t01
- ⛔ https://calibration.filscan.io/tx/bafy2bzaceayqt5cfkcgmse4ubezbfh2nl2336tmklengcxohtf5mrynalqoum

Also I'm working on the Glif tools, so would like to know if there is anywhere we can do better!

**Comment by tmm** [comment]
My mistake, thought it didn't work. Feel free to submit a new PR!

**Pull request merged**